metrics/database_handler.py

```diff
@@ -0,0 +1,89 @@
+"""
+Module containing database handler class.
```

Comment:
Is this file a copy from `common/` directory?

I'd rather include common/*.py in Dockerfile instead of duplicating code:

> ADD /common/*.py               /metrics/common/

Reply:
Its a copy, but it has the logging stuff removed.  I was intending to go back after this is released and modify the common/database_handler.py so the logger can be optional (use dependency injection or something similar)

Reply:
ah ok, or you could set the logging level to some high value to make sure it'll not print anything (logging.CRITICAL ?)

Reply:
I was under the impression that using that logging component also intercepted stuff printed to stdout.  In the current incarnation of the code, that's how it reports the metrics it collects.

Am I mistaken about the logging component grabbing the stuff printed to stdout?

In any case, it may be less of an issue once I modify the code to write the metrics to a file and then upload to S3 bucket.

Reply:
I'm not sure if it's like that but I saw this behaviour somewhere already. Nevermind, lets keep it as is now and when you move produced data to file, we can switch to regular database handler file and logging.